### ISSUE TITLE
How to reproduce the result on the paper？

### ISSUE THREAD

**Issue by ljjcoder** [WangYueFt/rfs#4]
Thanks for your excellent work！I encountered some problems when I wanted to reproduce your work on miniimagenet.
first I run the command:
CUDA_VISIBLE_DEVICES=0 python train_supervised.py --trial pretrain --model_path ./checkpoint --tb_path ./tensorboard --data_root ./mini
 then I run next command:
CUDA_VISIBLE_DEVICES=2 python train_distillation.py -r 0.5 -a 1.0 --path_t ./checkpoint/resnet12_miniImageNet_lr_0.05_decay_0.0005_trans_A_trial_pretrain/ckpt_epoch_90.pth \
--trial born1 --model_path ./checkpoint --tb_path ./tensorboard --data_root ./mini
I use the model trained by distill and run it 3 times:
i get the accuracy on 1-shot：
![1589338669](https://user-images.githubusercontent.com/40750189/81766412-a5abd400-9508-11ea-9a20-d5516dffc0fa.png)
i get the accuracy on 5-shot：
![1589338723(1)](https://user-images.githubusercontent.com/40750189/81766471-c70cc000-9508-11ea-91d7-0c3381b46849.jpg)
Although this is a very good result, it is still lower than the result of the paper：1-shot：64.82 5-shot：82.14

**Comment by HobbitLong** [comment]
You need to run at least 2 generations of distillation to check the results, please see Figure 4 and texts in the paper.

I think I used the 100 epoch model as teacher, instead of 90 epoch.
Besides, please replace "-r 0.5 -a 1.0" with "-r 0.5 -a 0.5", though the difference should be negligible. What I get is (for the first generation of distillation): 
64
64.42
63.83
